lib/ClimaCoreTempestRemap/src/onlineremap.jl

```diff
@@ -68,6 +70,83 @@ function remap!(
     return target
 end
 
+# This version of this function is used for distributed remapping
+function remap!(
+    target::Fields.Field,
+    R::LinearMap,
+    source::Fields.Field,
+    context::ClimaComms.AbstractCommsContext,
+    target_mesh::Meshes.AbstractMesh{2},
+    space_o,
+)
+    pid = ClimaComms.mypid(context)
+    nprocs = ClimaComms.nprocs(context)
+
+    target_array = parent(target)
+    source_array = parent(source)
```

Comment:
For now you are assuming that `source` is field on a singleton comms context: add an assertion and a comment to make clear this is what is happening.